### PR TITLE
Add comment parameter to loadbalancer_virtual_server

### PR DESCRIPTION
**Pull request by tmarshall98**
Add loadbalancer_virtual_server optional comment parameter

### CODE REVIEW

**Comment by tmarshall98** [comment]
Done

File: fortiadc/resource_fortiadc_loadbalance_virtual_server.go [hunk 1]
```diff
@@ -412,6 +419,7 @@ func resourceFortiadcLoadbalanceVirtualServerUpdate(d *schema.ResourceData, m in
 		ErrorPage:            d.Get("error_page").(string),
 		TrafficLog:           boolToEnable(d.Get("traffic_log").(bool)),
 		TransRateLimit:       fmt.Sprintf("%d", d.Get("transaction_rate_limit").(int)),
+		Comments:             d.Get("comments").(string),
```
Comment (pablo-ruth) [inline fortiadc/resource_fortiadc_loadbalance_virtual_server.go:422]: Typo here, the arg name is "comment"

File: docs/resources/loadbalance_virtual_server.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ resource "fortiadc_loadbalance_virtual_server" "myvirtualserver" {
 * `error_page` - (Optional) Error page on backend failure.
 * `persistence` - (Optional) Persistence configuration.
 * `transaction_rate_limit` - (Optional) Limit of transactions per second. Defaults to `0`.
+* `comments` - (Optional) Set the comment field on the resource.
```
Comment (pablo-ruth) [inline docs/resources/loadbalance_virtual_server.md:59]: same little typo here

**Comment by pablo-ruth** [comment]
There are differences in naming between docs, args and calls of args, where it's sometimes "comment" and other times "comments". Can you align all of them, probably to "comments" since it's the name of the field in the Fortiadc API ?

File: fortiadc/resource_fortiadc_loadbalance_virtual_server.go [hunk 3]
```diff
@@ -26,6 +26,11 @@ func resourceFortiadcLoadbalanceVirtualServer() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"comment": {
```
Comment (pablo-ruth) [inline fortiadc/resource_fortiadc_loadbalance_virtual_server.go:29]: I think we can go for "comments"

File: fortiadc/resource_fortiadc_loadbalance_virtual_server.go [hunk 4]
```diff
@@ -227,6 +232,7 @@ func resourceFortiadcLoadbalanceVirtualServerCreate(d *schema.ResourceData, m in
 		ErrorPage:            d.Get("error_page").(string),
 		TrafficLog:           boolToEnable(d.Get("traffic_log").(bool)),
 		TransRateLimit:       fmt.Sprintf("%d", d.Get("transaction_rate_limit").(int)),
+		Comments:             d.Get("comment").(string),
```
Comment (pablo-ruth) [inline fortiadc/resource_fortiadc_loadbalance_virtual_server.go:235]: I think we can go for "comments"

**Comment by pablo-ruth** [comment]
LGTM :)

**Comment by pablo-ruth** [comment]
I just released [v0.4.0](https://github.com/Ouest-France/terraform-provider-fortiadc/releases/tag/v0.4.0)  of this provider with your two PR. Thanks for your contributions @tmarshall98 !

**Pull request merged**